src/Lykke.Job.TransactionHandler.Services/MarginTrading/MarginDataService.cs

```diff
@@ -169,13 +37,30 @@ public MarginDataService(MarginDataServiceSettings settings)
                 Amount = amount,
                 PaymentType = paymentType.ConvertToDto()
             };
-            var result = await Api.ApiBackofficeMarginTradingAccountsDepositPostWithHttpMessagesAsync(_settings.ApiKey,
-                request);
 
-            return result.Body == true;
+            try
+            {
+                var result = await Api.ApiBackofficeMarginTradingAccountsDepositPostWithHttpMessagesAsync(
+                    _settings.ApiKey,
+                    request);
+
+                if (result.Body == true)
+                    return OperationResult.Success();
+                
+                return OperationResult.Error("Error deposit to margin account");
+            }
+            catch (HttpOperationException e)
+            {
+                return GetOperationResult(e.Response.Content);
+            }
+            catch (Exception)
+            {
+                return OperationResult.Error("Unknown margin trading error");
```

Comment:
Since it's unknown exception, better to not catch it and let it propagate over the call stack. It can be transient infrastructure failure or programming error, so swallow it - it's not good.